tls/s2n_security_policies.c

```diff
@@ -40,6 +40,14 @@ const struct s2n_security_policy security_policy_20190801 = {
     .ecc_preferences = &s2n_ecc_preferences_20200310,
 };
 
+const struct s2n_security_policy security_policy_20200518 = {
```

Comment:
This ended up kind of strange. the 2019 policy includes ecc preferences that didn't exist until 2020, and the 2020 policy includes 2014 policies. These should have been reversed. Additionally, SHOULD security_policy_20190801 have included the new curve? Will that break customers who were using cipher_preferences_20190801 before we merged it with ecc_preferences?

This definitely requires some sort of comment at least :\

Reply:
I added this security policy as I required a TLS1.3 security policy with ecc_curves: p-256 and p-384 only for testing. We currently don't have this combination. I can add comments to explain.

Reply:
But didn't customers WANT that combination? A big ask was that they didn't want the new curve by default.

Reply:
We wanted this combination for TLS1.2, I am adding this combination for TLS1.3 now. The default for TLS1.3 as requested by customers, was to include x25519 as the first curve.